source/index.html.md

```diff
@@ -569,6 +569,70 @@ fetch("https://api.gigapay.se/v2/payouts/?employee=12&accepted_at_null=True", {
     ]
 }
 ```
+
+> Request to retrieve all employees with string "skoog" in name, email or phone number:
+
+```python
+import requests
+
+response = requests.get(
+    'https://api.gigapay.se/v2/employees/?search=skoog',
+    headers={
```

Comment:
An example for the payout endpoint would be nice as well.

Reply:
That is the current functionality. I search on employee information. It's a bit unintuitive, but I don't know what would be more reasonable search parameters. In any case I will update documentation so its more understandable.

Reply:
@AlbinLindskog 
Example is now written, but since payouts only return employee_id. It is quite confusing for the reader of the documentation. 

You search for a payout, and in the response your search word is absent.

Any idea on how to make it better?

Reply:
You could write your example using ?expand=employee, then the employee info will be included in the request.

Reply:
Sounds great